### PR TITLE
Integration with Discogs API

### PR DESCRIPTION
**Pull request by RRNunes**
Integration with Discogs API

### CODE REVIEW

File: src/main/resources/application-dev.yml [hunk 1]
```diff
@@ -18,3 +18,12 @@ spring:
       settings:
         trace: false
         web-allow-others: false
+  cloud:
+
```
Comment (gfreivasc) [inline src/main/resources/application-dev.yml:22]: Remove this empty line

Comment (RRNunes) [reply src/main/resources/application-dev.yml:22]: Resolved

File: src/main/java/com/betanunes/vinylshelf/controller/VinylController.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    private static final int PAGE_SIZE = 20;
+    private final DiscogsClient discogsClient;
+    public VinylController(DiscogsClient discogsClient) {
+        this.discogsClient = discogsClient;
+    }
+
+    @SuppressWarnings("checkstyle:MagicNumber")
+    @GetMapping("/search")
+    public ObjectNode search(@RequestParam("q") String query)  {
+        return discogsClient.search(query, "master", 1, PAGE_SIZE);
```
Comment (gfreivasc) [inline src/main/java/com/betanunes/vinylshelf/controller/VinylController.java:20]: `"page"` must be a query parameter of the request.

```suggestion
    public ObjectNode search(@RequestParam("q") String query,
                             @RequestParam("page", required = false, defaultValue = "1") String page
    )  {
        return discogsClient.search(query, "master", page, PAGE_SIZE);
```

**Comment by gfreivasc** [comment]
Lgtm

**Review by gfreivasc — APPROVED** [review]
Lgtm

**Pull request merged**